src/GraphQL/ObjectExtensions.cs

```diff
@@ -247,7 +247,7 @@ public static object GetPropertyValue(this object propertyValue, Type fieldType,
 
                 if (!IsDefinedEnumValue(fieldType, value))
                 {
-                    throw new ExecutionError($"Unknown value '{value}' for enum '{fieldType.Name}'.");
+                    throw new InvalidOperationException($"Unknown value '{value}' for enum '{fieldType.Name}'.");
```

Comment:
This function only executes when GetArgument is called, and even then, for properly validated schema it shouldn't fail this check.  Plus, a bug within `IsDefinedEnumValue` prevented this line of code from ever running.  I did fix the bug, however, in this PR.